nvdaHelper/cppjieba/sconscript

```diff
@@ -0,0 +1,55 @@
+# A part of NonVisual Desktop Access (NVDA)
```

Comment:
same here in regards to https://github.com/nvaccess/nvda/blob/master/projectDocs/dev/copyrightHeaders.md